packages/compass-workspaces/src/provider.tsx

```diff
@@ -217,6 +228,11 @@ const noopWorkspacesService = {
     throwIfNotTestEnv();
     return null;
   },
+  loadSavedWorkspaces: async () => {
+    throwIfNotTestEnv();
+    return Promise.reject();
+  },
+  restoreSavedWorkspaces: () => throwIfNotTestEnv,
```

Comment:
This is a public interface of workspaces that you are adding this to, loading / restoring workspaces shouldn't be exposed through it: we shouldn't allow other parts of the app to trigger this process